.github/workflows/coverity.yml

```
@@ -0,0 +1,20 @@
+name: coverity
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  run-coverity:
+    runs-on: ubuntu-latest
+    if: github.repository == 'AIDASoft/podio'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cvmfs-contrib/github-action-cvmfs@v2
+    - uses: aidasoft/run-lcg-view@v2
+      with:
+        coverity-cmake-command: 'cmake -DCMAKE_CXX_STANDARD=17 ..'
```

```suggestion
        coverity-cmake-command: 'cmake -DCMAKE_CXX_STANDARD=17  -DENABLE_SIO=ON ..'
```
This should work with LCG_99